lib/fwup/stream.ex

```diff
@@ -52,27 +52,27 @@ defmodule Fwup.Stream do
     {:noreply, state}
   end
 
-  def handle_info({_port, {:data, <<"ER", code::16, message::binary>>}}, state) do
-    dispatch(state, {:error, code, message})
-    {:noreply, state}
-  end
-
   def handle_info({_port, {:data, <<"WN", code::16, message::binary>>}}, state) do
     dispatch(state, {:warning, code, message})
     {:noreply, state}
   end
 
-  def handle_info({_port, {:data, <<"OK", code::16, message::binary>>}}, state) do
-    dispatch(state, {:ok, code, message})
-    {:noreply, state}
+  def handle_info({port, {:data, <<"ER", code::16, message::binary>>}}, state) do
+    dispatch(state, {:error, code, message})
+    _ = Port.command(port, "ER")
```

Comment:
I double-checked the fwup code. I told you wrong, nothing needs to be sent. The port just needs to be closed. You can delete lines 62 and 69.